### DISCUSSION TITLE
Poster image not visible

### DISCUSSION THREAD

**Discussion by DanieleMagri** [google/model-viewer#3591] [Q&A]
I've never had problems with posters, but in this case I can't see some images, the format is always PNG, any idea what it could be? Some work, some don't

**Comment by DanieleMagri** [comment]
`      <model-viewer  bounds="tight" src="/assets/GLTFmodels/model.glb" ar ar-modes="scene-viewer quick-look webxr"
        environment-image="neutral" style="--poster-color:transparent ;" shadow-intensity="1" reveal="manual" poster="/assets/category/thumbnails/image.png" 
        >
        <button slot="ar-button" id="ar-button">
        AR
        </button>
        <div class="progress-bar" slot="progress-bar"></div>
      </model-viewer>`


CSS

`  model-viewer {
    display: flex;
    float: left;
    width: 40%;
    height: 100%;
  }`

**Reply by DanieleMagri** [reply]
The absurd thing is that if I have a **working** image, and I move it to a different folder, after obviously changing the url in the code, this doesn't work! And if I try to move the faulty images in the "not working" folder to the "working" one, I change the url, and they still don't work ...

**Reply by elalish** [reply]
From your description I might guess a server caching problem? See if you can reproduce it on a Glitch and link it here.

**Reply by DanieleMagri** [reply]
i'm testing the web app locally, i'll try to play a little with the browser cache

**Comment by mate-panyeczky** [comment]
any solution on this?
